src/main/java/com/github/gilday/darkmode/DarkModeDetector.java

```diff
@@ -4,14 +4,14 @@
 import java.io.IOException;
 import java.io.InputStreamReader;
 
-/** Queries the environment to see if macOS Dark Mode is enabled */
+/** Queries the environment to see if Dark Mode is enabled */
 final class DarkModeDetector {
 
   /**
-   * @return true if the user has enabled macOS Dark Mode
-   * @throws IllegalStateException when called on not-macOS
+   * @return true if the user has enabled Dark Mode
+   * @throws IllegalStateException when called on Linux
```

Comment:
but not on Windows? This should probably remain "when called on not-macOS", or we should generalize it to "when fails to read dark mode setting"

Reply:
Changed to: "... when plugin fails to read system dark mode setting. "

Used "system" to clarify that we talk about the system's settings.